### DISCUSSION TITLE
Not the same results in plugin scan and native scan

### DISCUSSION THREAD

**Discussion by brankoenero** [capacitor-community/bluetooth-le#481] [Q&A] [answered]
OS: Android 10
Capacitor: 4
Plugin: 2.1.0

Hi, I'm retrieving a lot of results when I execute requestLEScan, but I don't retrieve some devices that in the native scan I do and I don't know what is the reason.

I'm using the requestLEScan method sending an empty RequestBleDeviceOptions object. So I understand that is not a filter thing. Specifically I'm trying to detect an iMZ220 Zebra printer and an Apple AirPods (they are not the priority but the printer).

To show to my self that the printer is not found, I got the device id by using another app and I tried to connect to it without a success result. It seems like the plugin is not able to detect it or connect to it. Both devices was previously paired to the android device successfully.

Any idea?

Thanks.

**Comment by pwespi** [comment, marked as answer]
This plugin only supports Bluetooth **Low Energy** (BLE), and not Bluetooth Classic. Printers and headphones typically use Bluetooth Classic as far as I know, so they are not detected by the plugin.

**Reply by brankoenero** [reply]
Wow, Its true, it is because the Bluetooth protocol. 

Thanks @pwespi.

Obviously, there's no flag to connect to Classic Bluetooth, isn't there?

**Reply by pwespi** [reply]
No, there is no flag. The whole Bluetooth Classic functionality is not implemented in this plugin.
